docs: update installation guide (#11160)
apache/apisix: docs/en/latest/installation-guide.md

   apisix start -c <path to config file>
   ```

> APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured. Generally, APISIX gets installed at `/usr/local/apisix/` directory, so your configuration file will be present at `/usr/local/apisix/conf/` path.

Reviewer: ```suggestion
APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured. Generally, APISIX gets installed at `/usr/local/apisix/` directory, so the configuration file will be present at `/usr/local/apisix/conf/`.
```

> If you are unable to start APISIX and observe `Bind address already in use` in logs, it means the ports APISIX wants to use are already occupied by other running processes. Upon reviewing the use of these processes, you may decide to kill these processes or configure APISIX to use other non-conflicting ports.

Reviewer: I think this is pretty basic stuff, we can remove this.

> Other way is to configure the default listening port to other available port on your local machine.

Reviewer: ditto

> In case you get the Port binding logs, that says `Bind address already in use` when trying to run the APISIX server using `apisix start` command, it is most likely that the ports are being used by certain processes running on your machine. Try killing the process which is using the port. You can find the PID of the process that is using the process by running the `sudo netstat -tulpn` command and later using `kill <process id>` command to kill the process.

Reviewer 2: ```suggestion
If you are unable to start APISIX and observe `Bind address already in use` in logs, it means the ports APISIX wants to use are already occupied by other running processes. Upon reviewing the use of these processes, you may decide to kill these processes or configure APISIX to use other non-conflicting ports.
```

1. Some suggestion to reduce verbosity but you can edit again. If this is accepted, the sentence below needs to be removed as well.
2. You can add `netstat` back but note that flags can be distro / os specific (e.g. on mac `netstat: n: unknown or uninstrumented protocol`). Also `sudo` is not needed. If the audience don't have the needed privilege, they can escalate it themselves.

   apisix start -c <path to config file>
   ```

APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured. Generally, APISIX gets installed at `/usr/local/apisix/` directory, so your configuration file will be present at `/usr/local/apisix/conf/` path.

> In case you get the Port binding logs when trying to run the APISIX server using `apisix start` command, it is most likely that the ports are being used by certain processes running on your machine. Try deleting the process using the port or configure the default listening port to other available port on your local machine.

Reviewer 2: What `Port binding logs`? Be specific of the error

Reviewer 2: `deleting the process` --> `killing the process`

Author: done!

> In order to configure the default listening port to be `8000` without changing other configurations, your configuration file could look like this:

Reviewer 2: Why the change here? `For example` + `could` sound like a pair providing a sample config, but now the instruction sounds definitive, which it should not be.

Author: If you read the sentence above it that mentions `configure the default listening port to other available port on your local machine.`... this makes sense. We're showing how do you perform the last step mentioned in the documentation rather than giving you an example which IMO is vague at this point.

Reviewer 2: then `could` --> `should`

```suggestion
In order to configure the default listening port to be `8000` without changing other configurations, your configuration file should look like this:
```
